packages/path_provider/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.4.6
```

Comment:
nit: This is adding a feature, so IMO it should be a MINOR upgrade instead of a PATCH one. In the other PRs incrementing this decimal place is still considered a "MINOR" upgrade by pub because of how it handles semver for packages that are below >1.0.0. It treats them as `0.MAJOR.MINOR+PATCH` then.

https://semver.org/#spec-item-7

Reply:
Done